node/node.go

```diff
@@ -310,12 +309,12 @@ func logNodeStartupInfo(state sm.State, pubKey crypto.PubKey, logger, consensusL
 	}
 }
 
-func onlyValidatorIsUs(state sm.State, pubKey crypto.PubKey) bool {
-	if state.Validators.Size() > 1 {
-		return false
+func validatorHasMoreThan13Power(state sm.State, pubKey crypto.PubKey) bool {
+	_, val := state.Validators.GetByAddress(pubKey.Address())
+	if val != nil {
+		return val.VotingPower > (state.Validators.TotalVotingPower() / 3)
```

Comment:
Do we need better precision here, i.e. casting to `float64`? This will use flooring division, so it may give incorrect results in certain cases - should probably use the same convention here as we do during consensus, whatever it does.

Reply:
We don't use floor in consensus https://github.com/tendermint/tendermint/blob/50117be81a9290378a03c9c269b9e9b5fbb39b1d/types/validator_set.go#L682